### PR TITLE
feat: updates and comments to team agreement

### PR DESCRIPTION
**Pull request by VB-at-Bis**
### Description

Updated Team Agreement and discussion points (marked as [TBD]- to Be Discussed)

### CODE REVIEW

File: docs/Team_Agreements.md [hunk 1]
```diff
@@ -13,16 +13,33 @@
 - Add the Epic
 - Add the Sprint, if needed
 
+### Grooming cards
```
Comment (matthieu-foucault) [inline docs/Team_Agreements.md:16]:
```suggestion
### Refining cards
```
Due to some connotations of the word "grooming", "backlog refinement" is now preferred

**Review by AntBush — CHANGES_REQUESTED** [review]
Some spelling errors. Additionally, I think some things like scheduling time for refinement in-sprint and a new planning-poker/estimation tool are part of the refined agreements. 

The bug likelihood/impact matrix is something we also want to implement (but idk if that goes here or somewhere else)

File: docs/Team_Agreements.md [hunk 2]
```diff
@@ -13,16 +13,33 @@
...
+
+- T-shirt sizing for new cards as a way to facilitate preliminary review / prioritization by PO
+- Estimating/Reviewing cards async before refinement/sprint planning
+- Card should pass Definition of Ready (DoR) before it can be moved as a strint candidate
```
[the remark this answers is not in the document]
Comment (VB-at-Bis) [reply docs/Team_Agreements.md]: fixed

**Review by matthieu-foucault — APPROVED** [review]
I renamed the file to `CONTRIBUTING.md`, following the github convention, which should make the contributing guidelines easily visible when creating issues/PRs

**Review by AntBush — APPROVED** [review]
LGTM

**Pull request merged**